frontend/src/components/InnerNavbar/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+/***** CONSTS *****/
+import './InnerNavbar.scss';
+
+/***** TYPE DEFINITIONS *****/
+type TSidebar = React.FC<{
+  children: React.ReactNode;
+}>
+
+export const Sidebar: TSidebar = ({ children }) => {
+  return(
```

Comment:
Also please put a space between `return` and `(` as well as adding new line at end of file.